packages/cdk/lib/roles/policies/nextflow-submit-job-batch-policy.ts

```diff
@@ -0,0 +1,24 @@
+import * as iam from "monocdk/aws-iam";
+
+export interface NextflowSubmitJobBatchPolicyProps {
+  account: string;
```

Comment:
passing the account and region around is a concerning sign to me, are we expecting those to ever not match the context in which the script runs? E.g. would we see account/region pair change over the runtime?